muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/communicator/CommunicatorController.java

```diff
@@ -337,4 +338,27 @@ public void delete(CommunicatorUserLabel communicatorUserLabel) {
     communicatorUserLabelDAO.delete(communicatorUserLabel);
   }
 
+  /**
+   * Cleans list of UserEntities so that there are no duplicates present. Returns the original list.
+   * 
+   * @param userEntities
+   * @return
+   */
+  public List<UserEntity> cleanDuplicateRecipients(List<UserEntity> userEntities) {
```

Comment:
Should return void to better indicate that the given list is the one that the method modifies.